libs/gpg45-evaluator/src/main/java/uk/gov/di/ipv/core/library/gpg45/Gpg45ProfileEvaluator.java

```diff
@@ -90,6 +90,35 @@ public Optional<JourneyResponse> getJourneyResponseForStoredContraIndicators(
         return Optional.of(JOURNEY_RESPONSE_PYI_NO_MATCH);
     }
 
+    public boolean checkCiScoreBreaching(ContraIndicators contraIndicators)
```

Comment:
A few things:
- I would phrase boolean methods to more clearly indicate which way round they are - in this case `isCiScoreBreaching`
- Do you think the `Gpg45ProfileEvaluator` is the right place for this? Could it also be on `UserIdentityService`? Once you rebase, you'll find that there is a very similar `breachingCiThreshold` method which you probably want to combine with :)

Reply:
I've rebased to use breachingCiThreshold since, without the logging (which I'm also keen to remove), it's the same as what I was using